templates/output.html.twig

```diff
@@ -31,8 +31,13 @@
                 </div>
             </div>
             <div class="form-group">
-                <label for="lang">{{ msg('language-code') }}</label>
-                <select id="lang" class="form-control" name="langs[]" multiple data-placeholder="{{ msg('langs-placeholder') }}">
+                <label id="optional-lang-label" for="lang" class="lang-label {% if engine == 'transkribus' %}hidden{% endif %}">
+                    {{ msg('language-code') }}
+                </label>
+                <label id="transkribus-lang-label" for="lang" class="lang-label {% if engine != 'transkribus' %}hidden{% endif %}">
+                    {{ msg('transkribus-language-code') }}
+                </label>
+                <select id="lang" class="form-control" name="langs[]" multiple data-placeholder="{{ msg('langs-placeholder') }}" {% if engine == 'transkribus' %}required=''{% endif %}>
```

Comment:
`required` doesn't need to take a value, it's a boolean attribute (same as `multiple` which is already here).